components/NavBar.js

```diff
@@ -18,7 +18,7 @@ const StyledNavItem = styled.a`
   text-decoration: none;
   position: relative;
   display: inline;
-  margin-left: 16px;
+  margin-left: 23px;
```

Comment:
It looks better to use something in the scale `2, 4, 8, 16, 32` for spacing values wherever possible. In this case, `32px`

Reply:
On second thought, instead of this `margin-left` maybe the `StyledNavItem`s should be enclosed in `Box`es and use `ml=[0, 4]` to make them responsive.

Reply:
I have added the `Box` here